i2c/26-i2c.js

```diff
@@ -1,12 +1,20 @@
 module.exports = function(RED) {
     "use strict";
-    var I2C = require("i2c-bus");
+    try {
```

Comment:
That would make sense !
Should I also use node.error instead of console.error if it's inside the node ? I still wanted to print the error so the user has an idea of why it didn't load.

Reply:
You can- though don't forget node.error takes two parameters.

Reply:
Done also,
Decided to move only the logging part in the node declaration to avoid having to manage the I2C init in every node.
I think the results are the same though:
No errors/message when nodes aren't present in the workspace.
I tested with node.error but decided to go back to console.error as it's pretty annoying to have errors in the debug sidebar at each deploy. The fact there is status now also helps as the user will know the nodes aren't active.